File: windowslib/src/main/resources/scala-native/windows/accCtrl/trusteeForm.c
#if defined(_WIN32) || defined(WIN32)
#define WIN32_LEAN_AND_MEAN
#include <accctrl.h>

int scalanative_trustee_is_sid() { return TRUSTEE_IS_SID; }
int scalanative_trustee_is_name() { return TRUSTEE_IS_NAME; }
int scalanative_trustee_bad_form() { return TRUSTEE_BAD_FORM; }
int scalanative_trustee_is_objects_and_sid() {
    return TRUSTEE_IS_OBJECTS_AND_SID;
}
int scalanative_trustee_is_objects_and_name() {
    return TRUSTEE_IS_OBJECTS_AND_NAME;
}

#endif // defined(_WIN32)

File: windowslib/src/main/scala/scala/scalanative/windows/accctrl/TrusteeForm.scala
package scala.scalanative.windows.accctrl

import scala.scalanative.unsafe.*

@extern
object TrusteeForm {
  @name("scalanative_trustee_is_sid")
  def TRUSTEE_IS_SID: TrusteeForm = extern
  @name("scalanative_trustee_is_name")
  def TRUSTEE_IS_NAME: TrusteeForm = extern
  @name("scalanative_trustee_bad_form")
  def TRUSTEE_BAD_FORM: TrusteeForm = extern
  @name("scalanative_trustee_is_objects_and_sid")
  def TRUSTEE_IS_OBJECTS_AND_SID: TrusteeForm = extern
  @name("scalanative_trustee_is_objects_and_name")
  def TRUSTEE_IS_OBJECTS_AND_NAME: TrusteeForm = extern
}
